site/content/blogs/apprunner-waf.en.md

```diff
@@ -0,0 +1,82 @@
+# Using WAF With AppRunner in Copilot
+
+Posted On: Feb 08, 2023
+
+**Siddharth Vohra**, Software Development Engineer, AWS App Runner
+
+You can now associate your AWS WAF WebACLs with your App Runner service - all in Copilot!
+
+AWS Web Application Firewall (WAF) helps you monitor requests that are forwarded to your web applications and allows you to control access to your content. To do this, you need to create WAF Web Application Control List (Web ACL) which contains many rule options for your application. You can add multiple rules and create a Web ACL. Once you’ve created a Web ACL through AWS WAF Console or the AWS CLI, copy and save the Web ACL ARN. Now follow the steps below.
+
+ Step 1: If you don’t already have an App Runner service created/deployed, run `copilot init` to create and configure an App Runner service. (Do not deploy it just yet!)  
+
+Step 2: Go to the service's directory (users/\<your username\>/copilot/\<your App Runner service name\>). If you don't have a folder called 'addons' in this directory already, create a new folder and name it 'addons'. Now, go to the AR-WAF Addon discussion (Hyperlink coming soon) and follow the steps.  Your folders would now look like this:  
```

Comment:
I agree that we don't need to drop the discussion link here. This blog post is great enough for users to follow - we don't want to break the flow.

We can add a line at the end of this blog post, but not in the middle:
```
We posted these step in our [GitHub "Show and tell" discussion section](link) as well! If you have any questions, feedbacks or requests that are related to AppRunner's WAF support, feel free to drop a comment there!
```

Reply:
That makes sense. Will update this.

Reply:
Done